pages/api/stories/index.ts

```diff
@@ -64,10 +64,6 @@ async function handlePost(req: NextApiRequest, res: NextApiResponse): Promise<vo
     // Strip empty values
     if (req.body[k] === '') {
       return acc
-    }
-    // Parse anonymous to boolean
-    else if (k === 'anonymous') {
-      acc[k] = JSON.parse(req.body[k])
```

Comment:
🔪

Reply:
This is in `lib/api/stories.ts` now, and it's more complicated because Typescript. Byeee!